README.md

```diff
@@ -415,6 +415,7 @@ export MASTER_KEY=$(openssl rand -hex 32)
    ```bash
    helm upgrade --install artifactory-ha jfrog/artifactory-ha \
         --set artifactory.joinKey=$JOIN_KEY \
+        --set artifactory.masterKey=$MASTER_KEY \
```

Comment:
There's no need to provide masterKey when upgrading Artifactory (join key is enough)